src/System.IO.FileSystem/tests/File/GetSetTimes.cs

```diff
@@ -108,6 +108,35 @@ public void SetLastWriteTimeTicks()
             Assert.True(firstFileTicks <= secondFileTicks, $"First File Ticks\t{firstFileTicks}\nSecond File Ticks\t{secondFileTicks}");
         }
 
+        [ConditionalFact(nameof(isNotHFS))] // OSX HFS driver format does not support nanosecond granularity.
+        public void SetUptoNanoseconds()
```

Comment:
This comment does not seem right On this test

Reply:
whts wrong here ?

Reply:
Misread..ignore